### DISCUSSION TITLE
Born in a Barn replacement

### DISCUSSION THREAD

**Discussion by CalaMariGold** [ACGaming/UniversalTweaks#212] [Bugfix Suggestions]
Would be nice to have this in UT instead of as a separate mod!

https://legacy.curseforge.com/minecraft/mc-mods/born-in-a-barn

**Comment by ACGaming** [comment]
This was merged by Forge natively, so it’s not needed in a current 1.12 Forge environment.

**Comment by CalaMariGold** [comment]
@ACGaming not sure how true this is but it's from the dev of born in a barn 
![image](https://github.com/ACGaming/UniversalTweaks/assets/57605926/d1fbec71-6fae-4f37-a9da-b5ec37d1e1f5)

**Reply by ACGaming** [reply]
I have decompiled the mod and found the same functionality within Forge. So if I am not fundamentally wrong here, the same fixes are already applied.
